development/github/pkg/client/types.go

```diff
@@ -0,0 +1,39 @@
+package client
+
+import (
+	"github.com/google/go-github/v40/github"
+)
+
+// IssueTransferredEvent represent GitHub.IssuesEvent for transferred action.
+// It adds support for NewIssue and NewRepository json keys from GitHub webhook.
+type IssueTransferredEvent struct {
+	github.IssuesEvent
+	Changes *TransferredChange `json:"changes,omitempty"`
+}
+
+func (i *IssueTransferredEvent) GetChanges() *TransferredChange {
+	if i == nil {
+		return nil
+	}
+	return i.Changes
+}
+
+func (t *TransferredChange) GetNewIssue() *github.Issue {
+	if t == nil {
+		return nil
+	}
+	return t.NewIssue
+}
+
+func (t *TransferredChange) GetNewRepository() *github.Repository {
+	if t == nil {
+		return nil
+	}
+	return t.NewRepository
+}
```

Comment:
For better readability those functions should be below the structure that implements them

Reply:
Moved. Thanks